### PR TITLE
feat(cf-component-toast): Initial commit of basic toast

### PR DESCRIPTION
**Pull request by jculvey**
Creates a basic toast component for displaying inline success and error messages.

In the future, perhaps this can be enhanced to be dismissible, and anchor to different parts of the screen. For now, its a humble colored box.

### CODE REVIEW

**Comment by jwineman** [comment]
Can someone take me off default reviewers please? :)

File: packages/cf-component-toast/example/basic/component.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Toast from 'cf-component-toast';
+
+class ToastComponent extends React.Component {
+  render() {
+    return (
+      <div>
+        <Toast type="success">
```
Comment (jwineman) [inline packages/cf-component-toast/example/basic/component.js]: image

File: packages/cf-component-toast/src/index.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { applyTheme } from 'cf-style-container';
+
+import ToastUnstyled from './Toast';
+import ToastTheme from './ToastTheme';
+
+const Toast = applyTheme(ToastUnstyled, ToastTheme);
+
+export { Toast as default, ToastUnstyled, ToastTheme };
```
Comment (tajo) [inline packages/cf-component-toast/src/index.js]: please change this to:

```js
export { Toast, ToastUnstyled, ToastTheme };
```

We want to be consistent across the library with migrated components.

File: packages/cf-style-container/test/__snapshots__/index.js.snap [hunk 3]
```diff
@@ -147,6 +147,8 @@ Object {
     },
     "color": Object {
       "apple": "#BD2527",
+      "apple10": "#f8e9e9",
```
Comment (manatarms) [inline packages/cf-style-container/test/__snapshots__/index.js.snap]: Sorry I missed this PR! 
I like this idea. What do you think of adding a comment about how the naming convention works. I'm assuming `apple10` is like saying `optacity(apple,0.1)`.

Comment (neynah) [reply packages/cf-style-container/test/__snapshots__/index.js.snap]: Yeah that's exactly what it means. +1 to adding a note about the naming.

**Pull request merged**